American Whitewater/AppDelegate.swift

```diff
@@ -0,0 +1,109 @@
+//
+//  AppDelegate.swift
+//  American Whitewater
+//
+//  Updated by David Nelson on 7/4/19.
+//  Copyright © 2019 American Whitewater. All rights reserved.
```

Comment:
You can remove the comments when you get a chance, since this is all open source.

Reply:
Removed